functests/test_resources/door_entry_story.yaml

```diff
@@ -14,7 +14,7 @@ steps:
       description: Create a door in Paris at Jitsuin offices.
     signature:
       attributes:
-        arc_display_name: Jitsuin Paris Front Door
+        arc_display_name: Jitsuin Paris Front Door Demo5
```

Comment:
Probably didn't want to commit this Demo5 thing. Maybe in general we should be encouraging arc_namespace for this anyway (?)

Reply:
probably best solved by sourceing arc_namespace from the environment - just haven't got to that part yet...